### PR TITLE
Hooks

### PR DESCRIPTION
**Pull request by TSHEPO-CLOUD**
**Refactoring with Hooks**

 - Refactor the code with hooks
 - Instead of using class-based components, I used  Hooks.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,82 +1,86 @@
...
+    <div className="calculator">
+      <p className="calculator-result">{display || '0'}</p>
+      <div className="calculator-buttons">
+        <button name="AC" type="button" onClick={handleClick} className="grey">
+          AC
+        </button>
+        <button name="+/-" type="button" onClick={handleClick} className="grey">
+          +/-
+        </button>
+        <button name="" type="button" onClick={handleClick} className="grey">
```
Comment (MahmoudBakr23) [inline src/components/Calculator.js]: - [ ] Kindly double-check on this button `%`, it's not outputting any results of an operation! 🙏 😄

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @TSHEPO-CLOUD 😃, 

Great job refactoring your code from class-based component to functional component using react hooks and ES6 features. However, there are still some changes to make in order to get the approval 🙏 😄 

- [ ] Please write a more descriptive PR headline instead of the original branch name

_Please see the comment down below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (DeliceLydia) [reply src/components/Calculator.js]: Well addressed this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @TSHEPO-CLOUD , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**